tests/models/test_modeling_common.py

```diff
@@ -664,6 +681,117 @@ def test_deprecated_kwargs(self):
                 " from `_deprecated_kwargs = [<deprecated_argument>]`"
             )
 
```

Comment:
Maybe add a reference to transformers code

Reply:
It was there, though in this place. Could you elaborate a bit more?